src/main/java/com/djrapitops/extension/FactionsUUIDExtension.java

```diff
@@ -35,7 +35,10 @@
 import com.massivecraft.factions.integration.Econ;
 import com.massivecraft.factions.perms.Relation;
 import com.massivecraft.factions.perms.Role;
+import com.massivecraft.factions.util.TL;
```

Comment:
Related to the other comment
```suggestion
import com.massivecraft.factions.util.TL;
import com.djrapitops.plan.extension.NotReadyException;
```